diffrax/brownian/path.py

```diff
@@ -30,9 +30,14 @@ class UnsafeBrownianPath(AbstractBrownianPath):
     interval, ignoring the correlation between samples exhibited in true Brownian
     motion. Hence the restrictions above. (They describe the general case for which the
     correlation structure isn't needed.)
+
+    Depending on the `levy_area` argument, this can also be used to generate Levy area.
+    `levy_area` can be "", "space-time" or "space-time-time".
+
     """
 
     shape: PyTree[jax.ShapeDtypeStruct] = eqx.field(static=True)
+    levy_area: str = eqx.field(static=True)
```

Comment:
The type annotation here should be `Literal["", "space-time", "space-time-time"]`. This helps to be much more precise than simply `str`.